similarity/calc_sim.py

```diff
@@ -0,0 +1,78 @@
+import nltk
+from nltk.corpus import stopwords
+import numpy as np
+import pandas as pd
+from sklearn.cluster import KMeans
+import string
+
+
+# Reads csv as input with Pandas DataFrame
+def read_data(filename):
+  data = pd.read_csv(filename, skipinitialspace=True)
+  return data
+
+# map each word to an index
+def extract_words(data):
+  bag_of_words = {}
+  nltk.download('stopwords')
+  stop_words = set(stopwords.words('english'))
+  index = 0
+  for about in data['Mission']:
+    # remove all punctuations
+    for char in about:
+      if char in string.punctuation:
+        about = about.replace(char, ' ')
```

Comment:
Always be careful with mutating what you're iterating over. Usually this can lead to very strange behavior.

The issue is that there's not really a right way to handle this and different languages could take different approaches (many will just throw an exception).